docs/version0.27.md

```diff
@@ -68,4 +68,8 @@ The [_JAVA_OPTIONS environment variable](cmdline_specifying.md) (with different
 
 Global lock reservation is now enabled by default. This is an optimization targeted towards more efficient handling of locking and unlocking Java&trade; objects. The older locking behavior can be restored via the `-XX:-GlobalLockReservation` option. See [-XX:[+|-]GlobalLockReservation](xxgloballockreservation.md) for more details.
 
+### Increase default OS stack size on PPC64 platforms
```

Comment:
Please change to "Increase in default operating system stack size on PPC64 platforms" (makes title more consistent than others and spells out OS)